src/modules/a11y/a11y.js

```diff
@@ -131,9 +131,10 @@ export default function A11y({ swiper, extendParams, on }) {
 
   function updatePagination() {
     const params = swiper.params.a11y;
-    if (hasPagination()) {
-      swiper.pagination.bullets.each((bulletEl) => {
-        const $bulletEl = $(bulletEl);
+    const doHasPagination = hasPagination();
```

Comment:
Maybe there is a sense to replace this line with just 
```
if (!hasPagination()) return;
```
I mean what is the point to to loop over `swiper.pagination.bullets` and do checks for pagination inside if `hasPagination()` return false?